### PR TITLE
Enable multiple extra_cert

### PR DESCRIPTION
**Pull request by mkumatag**
This will enable multiple entries in the extra_cert variable.

usage:

```json
"extra_cert": "k8s-cluster.ppc64le-cloud.org,651b7744-eu-gb.lb.appdomain.cloud,k8s-cluster-int.ppc64le-cloud.org"
```

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/15#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fk8s-ansible).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/k8s-ansible/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Rajalakshmi-Girish — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/15#" title="Author self-approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/15#pullrequestreview-666407053" title="LGTM">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fk8s-ansible).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/k8s-ansible/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
